lib/os/Kconfig

```diff
@@ -46,6 +46,12 @@ config SYS_HEAP_ALLOC_LOOPS
 	  keeps the maximum runtime at a tight bound so that the heap
 	  is useful in locked or ISR contexts.
 
+config SYS_HEAP_RUNTIME_STATS
+	bool "System heap runtime statistics"
+	default n
```

Comment:
Pedantry: "n" is the default state for a bool, you can remove this line.

Reply:
I will remove it, thanks.